app/routes/organen/orgaan/mandatarissen.js

```diff
@@ -77,4 +79,14 @@ export default class OrganenMandatarissenRoute extends Route {
 
     return queryParams;
   }
+
+  getFilteredMandatarissen(mandatarissen, params) {
+    let filteredMandatarissen = mandatarissen;
+    if (params.activeOnly) {
+      filteredMandatarissen = mandatarissen.filter((mandataris) =>
+        moment().isBetween(mandataris.foldedStart, mandataris.foldedEnd)
```

Comment:
This isBetween returns false if foldedEnd is not set. But we do want to see those. 
You can first check if foldedEnd is set en then do this check.

Reply:
or maybe better, use the isActive method on the mandataris model